### ISSUE TITLE
gdformat breaks code when long string is used

### ISSUE THREAD

**Issue by patrickelectric** [Scony/godot-gdscript-toolkit#104]
code:
```gdscript
const simple_water = preload(
	"LONG_LONG_RESOURCE_STRING"
)
```

error(13, 1): Error parsing expression, misplaced: '\n'

**Comment by Scony** [comment]
It's already fixed on `master` in #79. You can install master's version via `pip3 install git+https://github.com/Scony/godot-gdscript-toolkit.git`

**Comment by patrickelectric** [comment]
thanks!
